src/core/util/ElfReader.hpp

```diff
@@ -39,8 +78,8 @@ class ElfSymbol {
 	int getSymbolType() const { return m_symbol_type; }
 
 	bool isSection() const { return m_type == SECTION; }
```

Comment:
why

Reply:
Mein tabify hat auch Intra-Line Multi-spaces in tabs verwandelt. Das ist natürlich humbug....